### DISCUSSION TITLE
SQLite的文艺复兴 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#149] [Announcements]
# SQLite的文艺复兴 · BMPI

本文介绍史上最流行的数据库SQLite从进程内到分布式的的文艺复兴故事。

https://www.bmpi.dev/dev/renaissance-sqlite/

**Comment by madawei2699** [comment]
## 推荐文章

**Reply by madawei2699** [reply]
- [Store SQLite in Cloudflare Durable Objects](https://ma.rkusa.st/store-sqlite-in-cloudflare-durable-objects)
